src/styles/brackets.less

```diff
@@ -1392,11 +1372,11 @@ a, img {
     font-size: 14px;
     color: @bc-text;
     background: @bc-panel-bg;
-    overflow: visible;  // needed for .error popup
+    overflow: visible; // needed for .error popup
     padding: 5px 4px 4px 14px;
 
     -webkit-transform: translate(0, 0); // Prefix still required.
-    transition: -webkit-transform 66ms cubic-bezier(0, 0.62, 0.04, 0.99);
+    transition: transform 66ms cubic-bezier(0, 0.62, 0.04, 0.99);
```

Comment:
Ah, I see. In that case, I'll just revert it to `transition: -webkit-transform` until we got CEF 2171 on Linux, too.